lib/components/src/controls/Range.tsx

```diff
@@ -27,6 +27,13 @@ const RangeInput = styled.input(({ theme }) => ({
     cursor: 'pointer',
   },
 
+  // FIXME
```

Comment:
Would be nice to explain what needs to be fixed here.

Reply:
This isn't really part of this PR. There is a separate open ticket about this exact thing.